Silicon/CommonSocPkg/Library/PchSpiLib/PchSpiLib.c

```diff
@@ -57,6 +57,17 @@ DisableBiosWriteProtect (
   IN  UINTN         PchSpiBase
   )
 {
+  UINT8   BiosCtrl;
+
+  ///
+  /// By default BIT5 (EISS) is set which makes BIOS region not writable.
+  /// Clearing this BIT to be able to write to the BIOS region
+  ///
+  BiosCtrl = MmioRead8 (PchSpiBase + R_SPI_BCR);
+  if (((BiosCtrl & B_SPI_BCR_SMM_BWP) != 0) && ((BiosCtrl & B_SPI_BCR_BLE) == 0)) {
+    MmioAnd8(PchSpiBase + R_SPI_BCR, (UINT8)(~B_SPI_BCR_SMM_BWP));
+  }
+
   if ((MmioRead8 (PchSpiBase + R_SPI_BCR) & B_SPI_BCR_SMM_BWP) != 0) {

```

Comment:
also please help change line 71 to B_SPI_BCR_EISS.

Reply:
done